test/fixtures/cdb_geocoder.sql

```diff
@@ -101,6 +101,18 @@ RETURNS Geometry AS $$
   END
 $$ LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION cdb_mapzen_geocode_street_point(searchtext TEXT, city TEXT DEFAULT NULL, state_province TEXT DEFAULT NULL, country TEXT DEFAULT NULL)
```

Comment:
I've just realised that `cdb_mapzen_geocode_street_point` fixture is not used by `GeoreferenceCity` analysis, that uses `cdb_mapzen_geocode_namedplace_point`. The same for other providers. @iriberri, which one is the good one?